Dockerfile.assisted-installer-controller

```
@@ -11,12 +10,14 @@ COPY go.sum go.sum
 RUN go mod download
 
 COPY . .
-RUN make controller
+RUN TARGETPLATFORM=$TARGETPLATFORM make controller
 
 FROM quay.io/centos/centos:stream8
+ARG TARGETPLATFORM
 
 RUN yum -y install make gcc unzip wget curl rsync && yum clean all
+RUN case $TARGETPLATFORM in linux/amd64 | "") arch=x86_64 ;; linux/arm64) arch=aarch64 ;; *) echo Invalid platform $TARGETPLATFORM && exit 1 ;; esac ; \
+   curl -k https://mirror.openshift.com/pub/openshift-v4/$arch/clients/ocp/4.11.3/openshift-client-linux.tar.gz | tar xfz - -C /usr/bin oc && chmod +x /usr/bin/oc
```

```suggestion
   curl --retry 5 --silent https://mirror.openshift.com/pub/openshift-v4/$arch/clients/ocp/4.11.3/openshift-client-linux.tar.gz | tar xfz - -C /usr/bin oc && chmod +x /usr/bin/oc
```
removed `-k` because I'm not sure we want insecure connection.